sunpy/map/map_factory.py

```diff
@@ -238,9 +244,13 @@ def _parse_url(self, arg, **kwargs):
     def _parse_path(self, arg, **kwargs):
         return parse_path(arg, self._read_file, **kwargs)
 
+    @_parse_arg.register(list)
```

Comment:
I have started playing with this locally, but my main issue here is this.

Registering the `list` type is to broad (even though it should work under almost all circumstances), I would much prefer it if we flattened out the list to just be a whole bunch of `OpenFile` objects and then parsed those.

If you don't mind @wmoapl I will push a commit which does this for Map, but I haven't done it for TimeSeries yet.

Reply:
Yes, that makes sense. Thanks @Cadair

Reply:
Turns out I can't push to this PR, can you merge this PR: https://github.com/APL-HTM-S3-AWARE/sunpy/pull/13

Reply:
I have now updated the timeseries factory as well if you can merge this: https://github.com/APL-HTM-S3-AWARE/sunpy/pull/14